frontend/www/js/omegaup/teamsgroup/edit.ts

```diff
@@ -45,6 +69,129 @@ OmegaUp.on('ready', () => {
               })
               .catch(ui.apiError);
           },
+          'update-search-result-users': (query: string) => {
+            api.User.list({ query })
+              .then((data) => {
+                // Users previously invited to any team in the current teams
+                // group can not be added to another, so they should not be
+                // shown in the dropdown
+                const addedUsers = new Set(
+                  this.teamsMembers.map((user) => user.username),
+                );
+
+                this.searchResultUsers = data
+                  .filter((user) => !addedUsers.has(user.label))
+                  .map((user) => ({
+                    key: user.label,
```

Comment:
```suggestion
                    key: user.value,
```

uno de estos días quizás sería bueno cambiar todos los `value`s por `key`s para que esta confusión ya no exista.

Reply:
Listo